site/docs/how-tos/grpc.md

```diff
@@ -246,6 +246,15 @@ stage.mapUsingServiceAsyncBatched(bidiService,
 })
 ```
 
+If your batch takes close to or more than 1 s (including the network
```

Comment:
"If your batch takes **more than ~0.8 seconds** (including the network overhead), you should increase the value **of the** following properties so that **the clean shutdown succeeds**"